index.Rmd

```diff
@@ -126,7 +137,7 @@ You can explore the data here
 
 ```{r echo=FALSE, warning=FALSE}
 #load the NFL data from a csv
-teams <- read.csv("data/nfl_franchises.csv", stringsAsFactors=F)
```

Comment:
Use ''read_csv' function from 'readr' instead of 'read.csv' (base function) as it eliminates the need for 'strings as factors'